package.json

```
@@ -55,6 +55,6 @@
     "vitest": "^0.17.0"
```

@drwpow you probably want to bump the actual `vitest` dependency too
```suggestion
    "vitest": "^0.34.1"
```

---

Ah good suggestion ty